content/en/getting_started/support/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Getting started with Datadog Support
+kind: documentation
+further_reading:
+- link: "https://docs.datadoghq.com/agent/troubleshooting/"
+  tag: "Documentation"
+  text: "Agent Troubleshooting"
+---
+
+## Overview
+
+Datadog provides two primary channels for customers seeking support:
+   - [Opening a support ticket][1].
```

Comment:
I'm not sure why, but this link doesn't work for me in the staging preview (even though the link itself listed at the bottom of the file is correct).

Reply:
Thanks for flagging, resolved in [98fefe5].(https://github.com/DataDog/documentation/pull/17834/commits/98fefe52097bbf2d480e7a5c9d6af65712ee099f)